Packs/Core/Playbooks/playbook-Exchange_forwarding_rule_configured.yml

```diff
@@ -0,0 +1,1671 @@
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "37":
+    id: "37"
+    taskid: f6057cef-9260-4970-8b1d-88edb25b4059
+    type: title
+    task:
+      id: f6057cef-9260-4970-8b1d-88edb25b4059
+      version: -1
+      name: Soft Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "15"
+      - "20"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 750,
+          "y": 1300
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "38":
+    id: "38"
+    taskid: 3b4840c0-990a-42d1-835b-a3bc6e18f5ad
+    type: condition
+    task:
+      id: 3b4840c0-990a-42d1-835b-a3bc6e18f5ad
+      version: -1
+      name: Check user notification requirement
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      '#default#':
+      - "14"
+      "yes":
+      - "39"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: inputs.SendNotification
+            iscontext: true
+          right:
+            value:
+              simple: "true"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 910,
+          "y": 1760
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "39":
+    id: "39"
+    taskid: 49505509-b341-41dd-83e9-c80018d07814
+    type: regular
+    task:
+      id: 49505509-b341-41dd-83e9-c80018d07814
+      version: -1
+      name: Send the user a notification via email
+      description: Sends an email.
+      script: EWSO365|||send-mail
+      type: regular
+      iscommand: true
+      brand: EWSO365
+    nexttasks:
+      '#none#':
+      - "14"
+    scriptarguments:
+      body:
+        simple: |-
+          Hello,
+          As part of our ongoing security measures, we detected unusual activity associated with your mailbox. A forwarding address and associated rule were automatically removed from your account to protect your data and ensure the security of our systems.
+
+           If you did not set up these rules, we recommend reviewing your recent activity and updating your account password immediately. If you require assistance or further information, please contact our security team.
+
+          Thank you for your understanding and cooperation.
+      subject:
+        simple: Forwarding rule was removed from your mailbox
+      to:
+        simple: ${Core.OriginalAlert.raw_abioc.event.identity_name}
```

Comment:
Are you sure this field contains the user's email address?! can there be incidents where this field contains a name and not the email?

Reply:
I saw that this field is an email. But I will push a few jsons from customers again.